README.md

```diff
@@ -384,10 +384,87 @@ if (result.isIntervalResult()) {
     }
 }
 ```
+To perform a [Multi-Analysis](https://keen.io/docs/api/#multi-analysis), use the MultiAnalysis.Builder instead. An instance of MultiAnalysisResult will be returned unless Group By and/or Interval parameters are included, in which case the MultiAnalysisResult(s) will be nested inside an IntervalResult and/or a GroupByResult, just like any other QueryResult for other single analysis types when grouping/intervals are applied:
+
+``` java
+final MultiAnalysis multiAnalysis = new MultiAnalysis.Builder()
+        .withCollectionName("the_collection")
```

Comment:
Sure. I made the case with @baumatron that if we were bumping major versions anyway, withEventCollection() sounds to me like it would expect something of type "Collection" or something similar, whereas withCollectionName() seems to imply it expects a String, but certainly I'm not adamant about that, and withEventCollection() matches the 'event_collection' JSON property name. Also, if using an IDE with intellisense, that's not really an issue--though those same IDEs will also likely have refactoring tools that make changing the method name two clicks. More importantly, we weren't really sure if we should shoot for a major version change or just a minor one (as evident with some of the other changes under discussion below). It'll get changed back.